src/remoteConnector.ts

```diff
@@ -786,7 +787,7 @@ export default class RemoteConnector extends Disposable {
 
 		// Only use experiment for SaaS
 		const forceUseLocalApp = getServiceURL(params.gitpodHost) === 'https://gitpod.io'
-			? (await this.experiments.get<boolean>('gitpod.remote.useLocalApp', session.account.id))!
+			? (await this.experiments.get<boolean>('gitpod.remote.useLocalApp', session.account.id, { gitpodHost: params.gitpodHost }))!
```

Comment:
This `gitpodHost` is always `https://gitpod.io`

Reply:
for now, later we are going to remove `?`, but first we should release this PR to both stable and latest and see effect on gitpod.io